scio-extra/src/main/scala/com/spotify/scio/extra/sparkey/SparkeyIO.scala

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright 2023 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.extra.sparkey
+
+import java.lang.Math.floorMod
+import java.util.UUID
+import com.spotify.scio.coders.{BeamCoders, Coder}
+import com.spotify.scio.extra.sparkey.instances._
+import com.spotify.scio.io.{TapOf, TapT, TestIO}
+import com.spotify.scio.util.RemoteFileUtil
+import com.spotify.scio.values.SCollection
+import com.spotify.sparkey.{CompressionType, SparkeyReader}
+import org.apache.beam.sdk.io.FileSystems
+import org.apache.beam.sdk.io.fs.MoveOptions.StandardMoveOptions
+import org.apache.beam.sdk.io.fs.{EmptyMatchTreatment, ResourceId}
+import org.slf4j.LoggerFactory
+
+import scala.collection.mutable
+import scala.jdk.CollectionConverters._
+
+/** Special version of [[com.spotify.scio.io.ScioIO]] for use with sparkey methods. */
+case class SparkeyTestIO[T] private (path: String) extends TestIO[T] {
+  override val tapT: TapT.Aux[T, T] = TapOf[T]
+}
+
+object SparkeyIO {
+  @transient private lazy val logger = LoggerFactory.getLogger(this.getClass)
+
+  val DefaultNumShards: Short = 1
+  val DefaultSideInputNumShards: Short = 64
+  val DefaultCompressionType: CompressionType = CompressionType.NONE
+  val DefaultCompressionBlockSize: Int = 0
+
+  def apply(path: String): SparkeyTestIO[SparkeyReader] = SparkeyTestIO[SparkeyReader](path)
+  def output[K, V](path: String): SparkeyTestIO[(K, V)] = SparkeyTestIO[(K, V)](path)
```

Comment:
If we rename class to `SparkeyIO`, we can get rid of those 2

Reply:
See previous comment